src/screens/OtherProfile/index.js

```diff
@@ -177,7 +177,7 @@ const OtherProfile = () => {
   const {mappingColorFeed} = useCoreFeed();
   const isSignedMessageEnabled = dataMain.isSignedMessageEnabled ?? true;
   const isAnonimityEnabled = dataMain.isAnonMessageEnabled && isSignedMessageEnabled;
-
+  const {handleUpdateDiscoveryUser} = useDiscovery();
   React.useEffect(() => {
     setDMChat('');
   }, [isSignedMessageEnabled]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new hook `useDiscovery()` is being used and its `handleUpdateDiscoveryUser` function is being destructured. Ensure that this hook is properly defined and returns the expected function. Also, make sure that the dependencies of this hook are correctly managed.
```diff
- 
+ const {handleUpdateDiscoveryUser} = useDiscovery();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->